complement/pipeline.yml

```diff
@@ -73,6 +73,53 @@ steps:
         - exit_status: 128
           limit: 3
 
+  - command:
+      # Build the Synapse Workers docker image, which is located at:
+      # https://github.com/matrix-org/synapse/blob/master/docker/Docker-workers
+      # Note that this image is based on matrixdotorg/synapse:latest, which we're just
+      # pulling from docker hub
+      - wget https://github.com/matrix-org/synapse/archive/master.tar.gz
+      - tar -xzf master.tar.gz
```

Comment:
do we really need the whole of the synapse repo? If it's just the dockerfile you're after, just curl that?

Reply:
Unfortunately the dockerfile depends on additional files, named [docker/conf/worker.yaml.j2](https://github.com/matrix-org/synapse/blob/anoa/synapse_worker_docker/docker/conf/worker.yaml.j2) and [docker/configure_workers_and_start.py](https://github.com/matrix-org/synapse/blob/anoa/synapse_worker_docker/docker/configure_workers_and_start.py).

It's probably easier to download the whole repo instead of the three files individually (which may become more files in the future).

Reply:
fair enough.

Note that this incantation won't work until the dockerfile in question is present on synapse `master`.

Reply:
On which note:

> I have confirmed that this pipeline succeeds locally.

are you *sure*